core/jazz_environment-event-handler/index.js

```diff
@@ -263,18 +263,22 @@ var processEventInitialCommit = function (environmentPayload, configData, authTo
         rejectUnauthorized: false
       };
 
-      logger.info("svcPayload" + JSON.stringify(svcPayload));
-      request(svcPayload, function (error, response, body) {
-        if (response.statusCode === 200 && body && body.data) {
-          return resolve(null, body);
-        } else {
-          logger.error(`Error creating ${env} environment in catalog: ${JSON.stringify(response)}`);
-          return reject({
-            "error": `Error creating ${env} environment for ${environmentPayload.domain} "_" ${environmentPayload.service} in catalog`,
-            "details": response.body.message
-          });
-        }
-      });
+      if (environmentPayload.service === 'ui' && environmentPayload.domain === 'jazz') {
+        resolve();
+      } else {
```

Comment:
To avoid nested if else loops, I would just say `return resolve();`  and then remove the else altogether.

Reply:
done